src-ui.v3/cabal.project.local

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
This file should not have been committed. Would be good to edit the gitignore to have this in there.